Specs/Scene/Metadata3DTilesExtensionSpec.js

```diff
@@ -1,5 +1,5 @@
 import { Metadata3DTilesExtension } from "../../Source/Cesium.js";
-import MetadataSchema from "../../Source/Scene/MetadataSchema.js";
+import { MetadataSchema } from "../../Source/Cesium.js";
```

Comment:
Ever since the ES6 upgrade we've been inconsistent about this. I've been following the pattern of having each on the same line but I can fix it throughout the `Metadata****Spec.js` files